### PR TITLE
Avoid function names for IE11

### LINKED ISSUE #13528: amp-bind white-listed functions not working on IE
## What's the issue?
[white-listed functions](https://www.ampproject.org/docs/reference/components/amp-bind#white-listed-functions) of amp-bind do not work on IE (in this example, it was the function - String split)

## How do we reproduce the issue?

1. Open this in Internet Explorer (ex: IE 11) - https://codepen.io/anon/pen/VQrOOy
2. Change the selection to Option 2
3. See the console error in dev tools (screenshot attached)

## What browsers are affected?
IE

iebug

### PR DESCRIPTION
**Pull request by choumx**
Fixes #13528.

- Fixes issue in `amp-bind` and bans future usage.

/to @danielrozenberg

### CODE REVIEW

File: build-system/conformance-config.textproto [hunk 1]
```diff
@@ -248,3 +248,11 @@ requirement: {
   whitelist: 'src/service/cid-api.js'
   whitelist: 'src/service/cid-impl.js'
 }
+
+# Function
+
+requirement: {
+  type: BANNED_PROPERTY_CALL
+  error_message: 'Function.prototype.name is not allowed due to IE11 non-support.'
```
Comment (danielrozenberg) [inline build-system/conformance-config.textproto]: Do we usually explain the reasons why we ban specific properties? By shaming specific browsers? :stuck_out_tongue_closed_eyes:

Comment (choumx) [reply build-system/conformance-config.textproto]: I figure some context helps so we can un-ban if/when we drop IE11 support.

Comment (danielrozenberg) [reply build-system/conformance-config.textproto]: Makes sense.

**Comment by jridgewell** [comment]
I’m pretty sure I told you this when it was written. 😜

**Comment by choumx** [comment]
@jridgewell I think you did. :)

**Pull request merged**